x-pack/plugins/lens/public/utils.ts

```diff
@@ -14,8 +14,7 @@ export const desanitizeFilterContext = (
   const emptyTextValue = i18n.translate('xpack.lens.indexpattern.emptyTextColumnValue', {
     defaultMessage: '(empty)',
   });
```

Comment:
@ppisljar The bug in the data table is very likely related to this change - as the rest of the context isn't spreaded anymore, `negate` is lost.